pylearn2/sandbox/nlp/models/tests/test_mlp.py

```diff
@@ -0,0 +1,15 @@
+'''
+Created on Jan 17, 2015
+
+@author: Minh Ngoc Le
```

Comment:
Should I put `__maintainer__ = "LISA Lab"` and `__email__ = "pylearn-dev@googlegroups"`? I don't know what maintaining this piece of code would be like.